packages/subgraph/src/osx/pluginSetupProcessor.ts

```diff
@@ -30,9 +30,9 @@ export function handleInstallationPrepared(event: InstallationPrepared): void {
     return;
   }
   // Load or create a new entry for the this plugin using the generated installation ID.
```

Comment:
generatePluginInstallationEntityId != generatePluginEntityId in admin.ts

You won't be able to retrieve the plugin.

Use generatePluginEntityId for both pls - your test will fail check my PR for how to fix it